Week4/WEEK-4 SOLUTION/week-4 solution.ipynb

```diff
@@ -0,0 +1,407 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b2054568",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sqlite3 as sq\n",
+    "import datetime"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "9c19bae0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "conn = sq.connect(\"cms.db\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "0decf429",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cur = conn.cursor()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "5298010e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<sqlite3.Cursor at 0x267ab5e30c0>"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cur.execute(\"\"\"create table if not exists contact\n",
```

Comment:
use of tripal quotes ?

Reply:
Triple quotes are used to span multiple lines, and they also serve as docstrings.